hs_composite_resource/models.py

```diff
@@ -321,32 +372,21 @@ def delete_old_xml_files(folder=''):
             if istorage.exists(map_xml_file_full_path):
                 istorage.delete(map_xml_file_full_path)
 
-        # first check if the new_aggr_name is a folder path or file path
-        name, ext = os.path.splitext(new_aggr_name)
-        is_new_aggr_a_folder = ext == ''
+        # first check if the new_path is a folder path or file path
+        name, ext = os.path.splitext(new_path)
+        is_new_path_a_folder = ext == ''
```

Comment:
So we use whether a path has extension or not to differentiate whether it is a folder or file path. Does it mean we don't support files with no extension? ```os.path.isfile()``` is more solid on determining whether it is a file or folder, right?

Reply:
```os.path.isfile()``` checks if a file exists or not. We don't need that here.